integration-tests/google/src/test/java/org/apache/camel/quarkus/component/google/it/GoogleComponentsTest.java

```diff
@@ -103,7 +103,7 @@ public void testGoogleCalendarComponent() throws InterruptedException {
                 .statusCode(204);
 
         // Wait for calendar deletion to occur
-        Thread.sleep(500);
+        Thread.sleep(1000);
 
```

Comment:
Wouldn't some sort of await be better? Something like we do in telegram? https://github.com/apache/camel-quarkus/blob/master/integration-tests/telegram/src/test/java/org/apache/camel/quarkus/component/telegram/it/TelegramTest.java#L150-L161

Reply:
Yes, I can do a follow up fix for that.